src/com/lftechnology/java/training/srijan/basics/football/PointsStructure.java

```diff
@@ -0,0 +1,146 @@
+package com.lftechnology.java.training.srijan.basics.football;
+
+import java.util.logging.Logger;
+
+
+public class PointsStructure {
+	private static final int WIN;
+	private static final int LOSE;
+	private static final int DRAW;
+	private int points;
+	private int counts;
+	private int wins;
+	private int draws;
+	private int lose;
+
+	static {
+		WIN = 3;
+		LOSE = 0;
+		DRAW = 1;
+	}
+
+	public PointsStructure(String club) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "club". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1172)